src/js_native_api.h

```diff
@@ -531,7 +532,7 @@ NAPI_EXTERN napi_status napi_detach_arraybuffer(napi_env env,
 NAPI_EXTERN napi_status napi_is_detached_arraybuffer(napi_env env,
                                                      napi_value value,
                                                      bool* result);
-#endif  // NAPI_EXPERIMENTAL
+#endif  // NAPI_VERSION >= 6
```

Comment:
I don't see napi_is_detached_arraybuffer, napi_detach_arraybuffer in the 10.x staging branch and a quick look at PR's I don't see anything labelled with napi that would add them. 

Did we intend to add them to NAPI_VERSION 6?  I'm thinking no (unless I've missed the PR where they will be added)  since we were waiting for other PR to land in 10.x first.